DOCS.md

```diff
@@ -18,6 +18,7 @@ The following parameters are used to configure this plugin:
 * `vars` - variables to use in `template`
 * `secrets` - variables to use in `secret_template`. These are base64 encoded by the plugin.
 * `secrets_base64` - variables to use in `secret_template`. These should already be base64 encoded; the plugin will not do so.
+* `post` - commands to run after applying the Kubernetes files
```

Comment:
Should we name this `post_cmds` just to be more clear when reading the yaml files?